src/translations/enTranslations.json

```
@@ -690,8 +690,8 @@
       "files_uploaded_other": "{{count}} files are uploaded",
       "general_support_pending": "Pending follow-up",
       "get_curator_support": "Get help from a curator",
-      "imported_from_source": "",
-      "imported_from_source_and_archive": "",
+      "imported_from_source": "Imported from{{source}}",
```

Her bør vi gå inn i lokalise og legge inn en space, så lage en ny PR
```suggestion
      "imported_from_source": "Imported from {{source}}",
```